test/plz_run/test_repo/src/BUILD_FILE

```diff
@@ -15,4 +15,13 @@ sh_cmd(
 sh_cmd(
     name = "wobble",
     cmd = "sleep 0.1 && echo wobble",
-)
\ No newline at end of file
+)
+
+# Test add_data functionality
+build_rule(
+    name = "_add_data_rule",
+    test = True,
+    no_test_output = True,
+    test_cmd = "if [ $DATA != $PKG_DIR/dummydata.txt ]; then exit 1; else exit 0; fi;",
+    pre_build = lambda name: add_data(name, "dummydata.txt"),
```

Comment:
Would be nice to see this add a build target as data too.

Reply:
yeah +1 that is a significantly different codepath internally so would be more comfortable if we knew we covered that.

Reply:
Looks like there's another test below  that covers that now :)